README.md

```diff
@@ -58,11 +58,13 @@ Follow the instructions for your distribution:
 
 ## Building from source
 
-Before building, you will need the following
+### Requirements
+
+-	[Terraform](https://www.terraform.io/downloads.html) 0.11.x
```

Comment:
we have `0.12` in master so the 0.11 is wrong. I wouldn't add any version here since an user could also decide to use terraform 11 if he pick up the older releases etc.

Reply:
Got it removing it!